src/daft-core/src/array/fixed_size_list_array.rs

```diff
@@ -11,6 +11,7 @@ use crate::{
 #[derive(Clone, Debug)]
 pub struct FixedSizeListArray {
     pub field: Arc<Field>,
+    /// contains all the elements of the nested lists flattened into a single contiguous array.
```

Comment:
👍 

We're seriously lacking on some internal documentation. Feel free to add more as you explore the codebase!

Reply:
🥳